website/docs/components/accordion/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,102 @@
+## Usage
```

Comment:
One thing I think is missing is an indication if it's acceptable or not to mix&match in the same accordion different types of accordion item (with/without interactive elements in the toggle).
Something like this, is OK? 
<img width="805" alt="screenshot_2880" src="https://github.com/hashicorp/design-system/assets/686239/3310ef6d-0b9a-4c76-aaaa-ce7c4b7157fd">

(no strong opinions, but as a consumer I would have a doubt and it would be better to clarify our guidance on this)

Reply:
@cveigt

Reply:
Yes, this is a good point. I clarified our recommendation in the Generic content section. @KristinLBradley, could you update this example so that both containers `containsInteractive`?